pkg/logutil/sensitive.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logutil
+
+import (
+	"regexp"
+)
+
+var (
+	passwordPatterns = `(password: (\\")?)(.*?)((\\")?\\n)`
+	sslPatterns      = `(ssl-(ca|key|cert)-bytes:)((\\n\s{4}-\s\d+)+)`
+
+	beginPattern      = `(.*?)(-{5})BEGIN( [A-Z]+)+(-{5})`
```

Comment:
Maybe let some comments to explain that we try to match PEM format

https://en.wikipedia.org/wiki/Privacy-Enhanced_Mail